### PR TITLE
Updating the AIHWKIT  'Read The Doc' Paper references

### PR DESCRIPTION
**Pull request by soumyajeet8**
## Description

To update the AIHWKIT 'Read the Docs'

## Details

Updated the Paper References section, and also the steps to create a Basic Inferencing experiment using AIHWKIT

### CODE REVIEW

**Review by maljoras — APPROVED** [review]

**Review by maljoras — APPROVED** [review]

**Review by maljoras — APPROVED** [review]

File: docs/source/using_experiments.rst [hunk 1]
```diff
@@ -186,14 +231,23 @@ experiments locally::
     my_cloud_runner = CloudRunner()
     cloud_experiment = my_cloud_runner.run(my_experiment)
 
+Similarly ``Inferencing`` experiment can also be performed in the cloud by using 
+the ``InferenceCloudRunner`` , in an analogoud way as running experiments locally ::
```
Comment (kaoutar55) [inline docs/source/using_experiments.rst:235]: analogous

File: docs/source/using_experiments.rst [hunk 2]
```diff
@@ -42,13 +42,44 @@ An Experiment can be created just by creating an instance of its class::
...
+      inference_time = I_TIMES
     )
 
+
 Each Experiment has its own attributes, providing sensible defaults as needed.
 For example, the ``BasicTraining`` Experiment allows setting attributes that
 define the characteristics of the training (``dataset``, ``model``,
 ``batch_size``, ``loss_function``, ``epochs``, ``learning_rate``).
 
+Similarly the ``BasicInferencing`` Experiment allow setting attributes
```
Comment (kaoutar55) [inline docs/source/using_experiments.rst:79]: allows

File: docs/source/using_experiments.rst [hunk 3]
```diff
@@ -121,16 +165,17 @@ epoch::
      ...
     ]
 
-The local runner will also print information by default while the experiment
-is being executed (for example, if running the experiment in an interactive
-session, as a way of tracking progress). This can be turned off by the
+The ``LocalRunner`` for ``Training`` experimnet and ``InferenceLocalRunner`` for ``Inferencing`` experiment
```
Comment (kaoutar55) [inline docs/source/using_experiments.rst:168]: for  ``Training``  experiment

File: docs/source/using_experiments.rst (hunk 1 above)
Comment (kaoutar55) [inline docs/source/using_experiments.rst:235]: analogoud --> analogous

**Review by kaoutar55 — CHANGES_REQUESTED** [review]

**Comment by kaoutar55** [comment]
Thanks @soumyajeet8 please fix the minor typos

**Comment by kaoutar55** [comment]
Hi @soumyajeet8 did you get a chance to make the requested changed. They are minor changes.

**Review by kaoutar55 — APPROVED** [review]

**Comment by soumyajeet8** [comment]
HI @kaoutar55 , I have made the minor typo changes as requested since I was facing issue connecting to WSL so I had to resolve that, before I could access the code-base and make changes. The changes are captured in PR #568

**Pull request merged**